### DISCUSSION TITLE
v4.8.4

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#135] [Announcements]
### Adminer 4.8.4 (released 2024-04-07):

- Removed size for integer on MySQL 8+
- Replaced deprecated by error_get_last()
- Allow responsive styles on larger devices (PR https://github.com/adminerevo/adminerevo/pull/75)
- Fixed issue with editor tables list
- Adjusted href of editor title
- Fixes of warnings on PHP8.3
- Filter removal buttons on table select pages (PR https://github.com/adminerevo/adminerevo/pull/77)
- Skip dump of generated columns (PR https://github.com/adminerevo/adminerevo/pull/95)
- Add LICENSE file, clarify dual-licensing (PR https://github.com/adminerevo/adminerevo/pull/122)
- Restore latest version check (PR https://github.com/adminerevo/adminerevo/pull/129)

<hr /><em>This discussion was created from the release <a href='https://github.com/adminerevo/adminerevo/releases/tag/v4.8.4'>v4.8.4</a>.</em>

**Comment by skerbis** [comment]
Thx

**Comment by mckaygerhard** [comment]
one question.. still are possible to use this on older engines? mysql, postgresql.. etc ODBC? @LionelLaffineur

**Reply by mckaygerhard** [reply]
as pointed in comment https://github.com/adminerevo/adminerevo/issues/145#issuecomment-2173445590 adminer breaks mysql compatibility with older versions..

**Comment by imam-syaifudin** [comment]
its work thanks

**Comment by takizo** [comment]
Hey folks. I am seeing this error on this release. Previous version was ok, not sure anyone seeing this error. 
I am still running php 5.6.40 ;) 

![image](https://github.com/user-attachments/assets/fbfe614d-7129-440d-bb06-6226c2b2023b)

**Reply by mckaygerhard** [reply]
umm i also runs 5.6 i can check this weekend and you can ping me if i dont put any feedback..

**Comment by mckaygerhard** [comment]
ok i can confirm the error, it was using php 5.6.40 with alpine linux 3.8 @takizo also this newer versions breack mysql compatibility for older versions

```
[Thu Jan 09 17:49:44.033062 2025] [proxy_fcgi:error] [pid 4037:tid 3078651220] [client 10.10.34.160:42080] AH01071: Got error 'PHP message: PHP Parse error:  syntax error, unexpected '?' in /usr/share/webapps/adminer/adminer-4.8.4.php on line 65\n'
```

precisely on  `PHP Parse error:  syntax error, unexpected '?' in adminer/include/functions.inc.php on line 502\n'`

the associated issue is long well knowed as #83 and can be easyle fixed [i commented](https://github.com/adminerevo/adminerevo/issues/83#issuecomment-2581008324) on! cos **_if i cannot use adminer in older system we just use the others.._** **adminer is an alternative that still works in older and newer systems as well**

The fix was proposed at #154 about https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op but was reverted with new commits
